dbt_subprojects/daily_spellbook/models/_metrics/transactions/chains/ton/metrics_ton_transactions_daily.sql

```diff
@@ -0,0 +1,66 @@
+{{ config(
+        schema = 'metrics_ton'
+        , alias = 'transactions_daily'
+        , materialized = 'incremental'
+        , file_format = 'delta'
+        , incremental_strategy = 'merge'
+        , unique_key = ['blockchain', 'block_date']
+        , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+        )
+}}
+
+with ton_prices as ( -- get price of TON for each day to estimate USD value
+    select
+        date_trunc('day', minute) as price_day
+        , avg(price) as price
+    from {{ source('prices', 'usd') }}
+    where true
+        and symbol = 'TON'
```

Comment:
```suggestion
        and symbol = 'TON' and blockchain is null
```

Reply:
for consistency with fees model, and to be safe if other chains uses this symbol.

Reply:
fyi, we will eventually be cleaning up how we store and process native token prices. once we change that, we'll be sure to update all downstream spells usage on joins. for now, this remains fine.